Update readme, setup py and remove requirements-files (#14)
equinor/isar: README.md

- Send: The state machine has received a mission and is sending the mission to the robot.
- Monitor: The robot has received a mission from the state machine, and the state machine is monitoring the current
  mission.
- Collect: The state collects data during a mission.
- Cancel: The state machine has received a request to abort, or an event has occurred which requires the mission to be canceled. The cancel state also functions as a wrap-up state when a mission is finished, prior to the state machine returning to idle.

### Flask API

The Flask API establishes an interface to the state machine for the user. As the API and state machine are separate
threads, they communicate through python queues. The main flask extension used for the API design is
the [flask-restx](https://github.com/python-restx/flask-restx) package.

## Installation

```bash
$ pip install git+https://github.com/equinor/isar.git@main
```

Note, installation might require [wheel](https://pypi.org/project/wheel/).

> To run the API and statemachine, run:

Author: Agreed and done

```bash
$ python -m flask run --no-reload
```

## <a name="dev"></a>Development

> For local development, fork the repo, clone it and run in the root folder of the repository:

Reviewer: ```suggestion
For local development, please fork the repository. Then, clone and install in the repository root folder:
```

Author: Agreed and done
